Trust/Trust/Main/Home/HomeView.swift

```diff
@@ -27,6 +115,16 @@ struct HomeView: View {
                 }
             }
         }
+        .sheet(isPresented: $showCreateWalletView, onDismiss: {
+            showCreateWalletView = false
+        }) {
+            CreateWalletView()
+        }
+        .sheet(isPresented: $showAddWalletView, onDismiss: {
+            showAddWalletView = false
+        }) {
+            AddWalletView()
+        }
```

Comment:
In the Trust Wallet app, the create and add wallets buttons do not have a chevron. Typically, CTAs with chevron accessories push the view on the navigation stack, while plain buttons (without a chevron) present screens modally. I may be nitpicking. 😅 

Regardless of the design decision, I opted for the modal presentation just to show a different presentation style. :)